hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+function playHangman(){
+  console.log("welcome to hangman");
+  words = ['javascript','ruby','rails','jquery','programming'];
+  dashedWords = ['- - - - - - - - - -','- - - -','- - - - -','- - - - - -','- - - - - - - - - - -'];
+  guesses = [];
+  incorrectGuesses = 0;
+function randomizeIndex(){
+    randomIndex = Math.floor(Math.random()*5);
+  }
+randomizeIndex();
+}
+function randomWord(){
+  word = words[randomIndex];
```

Comment:
looks like we run into an issue with scope, we need to access `words` while in the context of `playHangman` where the variable is defined. Maybe we invoked this `randomWord` function in the `playHangman` function.